st2api/st2api/controllers/v1/webhooks.py

```diff
@@ -51,11 +51,12 @@ def __init__(self, *args, **kwargs):
         self._trigger_types = WEBHOOK_TRIGGER_TYPES.keys()
 
         self._trigger_dispatcher = TriggerDispatcher(LOG)
+        queue_suffix = self.__class__.__name__ + ':' + str(uuid.uuid4())
```

Comment:
I have not been able to find a restriction of queue name length. Likely that anything more than 256 would be rejected by rabbitmq server.

On the other hand and Q name more than 50 chars makes it harder to grok. I am +1 to some truncation :)